shapetree.ttl

```diff
@@ -0,0 +1,325 @@
+@prefix : <#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+
+
+<>
+  dcterms:issued "2020-07-01"^^xsd:date ;
+  dcterms:modified "2020-07-06"^^xsd:date ;
+  rdfs:label "ShapeTree terms"@en .
+
+
+# ShapeTree Class
+#############################################################################
+# Defines a tree of related resources which combine to enable some task
+# see: https://shapetrees.github.io/specification/primer
+#############################################################################
+:ShapeTree 
+  a rdfs:Class ;
+  rdfs:comment "Defines a tree of related resources which combine to enable some task"@en ;
+  rdfs:isDefinedBy <> ;
+  rdfs:label "ShapeTree"@en .
```

Comment:
When writing about shape trees in regular text I typically don't combine into one word (ShapeTree), but instead do Shape Tree or shape tree depending on whether its a heading or regular text.

Reply:
Updated.